docs/source/tutorials/pasha/pasha.rst

```diff
@@ -0,0 +1,74 @@
+PASHA: Efficient HPO and NAS with Progressive Resource Allocation
```

Comment:
AFAI can see, only two image files are needed here: pasha_illustration.png and soft_ranking.png. Please either remove the remaining files, or make use of them in the tutorial.

Reply:
yes, only those two files are used - I've removed the other ones already